core/src/test/java/io/questdb/test/griffin/AsOfJoinTest.java

```diff
@@ -1107,7 +1107,7 @@ public void testNestedASOF_keySymbol() throws Exception {
     }
 
     @Test
-    public void testNestedLT_keySymbol() throws Exception {
+    public void     testNestedLT_keySymbol() throws Exception {
```

Comment:
nit: the added space chars look like broken formatting

Reply:
fixed herehttps://github.com/questdb/questdb/pull/3399/commits/9f23069d36bd001529f82f91e2609128f7e2ceef